src/main/scala/org/openhorizon/exchangeapi/route/apikey/UserApiKeys.scala

```diff
@@ -0,0 +1,274 @@
+package org.openhorizon.exchangeapi.route.apikey
+
+import com.github.pjfanning.pekkohttpjackson.JacksonSupport
+import io.swagger.v3.oas.annotations._
+import io.swagger.v3.oas.annotations.enums.ParameterIn
+import io.swagger.v3.oas.annotations.media._
+import jakarta.ws.rs.{GET, POST, DELETE, Path}
+import org.apache.pekko.actor.ActorSystem
+import org.apache.pekko.event.LoggingAdapter
+import org.apache.pekko.http.scaladsl.model.{StatusCode, StatusCodes}
+import org.apache.pekko.http.scaladsl.server.Directives._
+import org.apache.pekko.http.scaladsl.server.Route
+import org.openhorizon.exchangeapi.auth._
+import org.openhorizon.exchangeapi.table.apikey.{ApiKeyMetadata, ApiKeysTQ, ApiKeyRow}
+import org.openhorizon.exchangeapi.table.user.{User, UserRow, UsersTQ}
+import org.openhorizon.exchangeapi.table.resourcechange.{ResChangeCategory, ResChangeOperation, ResChangeResource, ResourceChange, ResourceChangeRow, ResourceChangesTQ}
+import org.openhorizon.exchangeapi.utility._
+import java.util.UUID
+import slick.jdbc.PostgresProfile.api._
+import scala.concurrent.ExecutionContext
+import scala.util._
+import _root_.org.openhorizon.exchangeapi.utility.ApiKeyUtils
+import org.openhorizon.exchangeapi.utility.HttpCode
+import io.swagger.v3.oas.annotations.parameters.RequestBody
+import scala.concurrent.Future
+import scala.util.{Success, Failure}
+import java.sql.Timestamp
+import java.time.ZoneId
+
+@Path("/v1/orgs/{organization}/users/{username}/apikeys")
+@io.swagger.v3.oas.annotations.tags.Tag(name = "apikey")
+trait UserApiKeys extends JacksonSupport with AuthenticationSupport {
+
+  def db: Database
+  def system: ActorSystem
+  def logger: LoggingAdapter
+  implicit def executionContext: ExecutionContext
+ 
+  // === POST /v1/orgs/{organization}/users/{username}/apikeys ===
+@POST
+@Operation(
+  summary = "Create a new API key for a user",
+  description = "Creates a new API key for the specified user. Can be called by the user or org admin.",
+  parameters = Array(
+    new Parameter(name = "organization", in = ParameterIn.PATH, required = true, description = "Organization ID"),
+    new Parameter(name = "username", in = ParameterIn.PATH, required = true, description = "Username")
+  ),
+  requestBody = new RequestBody(
+    required = true,
+    content = Array(new Content(
+      mediaType = "application/json",
+      schema = new Schema(implementation = classOf[PostApiKeyRequest]),
+      examples = Array(
+        new ExampleObject(value = """{
+          "description": "Test API key for user"
+        }""")
+      )
+    ))
+  ),
+  responses = Array(
+    new responses.ApiResponse(responseCode = "201", description = "resource created - response body:",
+      content = Array(new Content(
+        mediaType = "application/json", 
+        schema = new Schema(implementation = classOf[PostApiKeyResponse]),
+        examples = Array(
+          new ExampleObject(value = """{
```

Comment:
These examples use pre-formatted text `""" ... """` denotes this. keep in mind the indentation/whitespace you add here will show in the rendered swagger doc. You will want to construct the examples off of the left edge.